Fix error in ephemeral deployment (and possibly staging)

During the smoke-tests phase:

`🗑️  Cleaning up monthly counters...
 ⛅️ wrangler 4.72.0
───────────────────
Resource location: remote 
🌀 Executing on remote database rushomon (9bfffd95-9202-4e1a-8ee4-8206fefd09a3):
🌀 To execute on your local development database, remove the --remote flag from your wrangler command.
✘ [ERROR] A request to the Cloudflare API (/accounts/***/d1/database/9bfffd95-9202-4e1a-8ee4-8206fefd09a3/query) failed.
  no such column: org_id at offset 35: SQLITE_ERROR [code: 7500]
  
  If you think this is a bug, please open an issue at: https://github.com/cloudflare/workers-sdk/issues/new/choose
🪵  Logs were written to "/home/runner/.config/.wrangler/logs/wrangler-2026-03-12_13-48-39_794.log"
🗑️  Cleaning up organizations...
 ⛅️ wrangler 4.72.0
───────────────────`

See https://github.com/piffio/rushomon/actions/runs/23004961980/job/66799124975?pr=159 for an example

--- a/.github/workflows/deploy-ephemeral.yml
+++ b/.github/workflows/deploy-ephemeral.yml
@@ -328,17 +328,27 @@ jobs:
 
           echo "🗑️  Cleaning up monthly counters..."
           wrangler d1 execute rushomon --remote -c wrangler.ephemeral.toml --command \
-            "DELETE FROM monthly_counters WHERE org_id = 'test-org-smoke'" || true
+            "DELETE FROM monthly_counters WHERE billing_account_id = 'test-org-smoke'" || true
 
           echo "🗑️  Cleaning up organizations..."
           wrangler d1 execute rushomon --remote -c wrangler.ephemeral.toml --command \
             "DELETE FROM organizations WHERE id = 'test-org-smoke'" || true
 
-          # Create test organization and user first
+          echo "🗑️  Cleaning up billing accounts..."
+          wrangler d1 execute rushomon --remote -c wrangler.ephemeral.toml --command \
+            "DELETE FROM billing_accounts WHERE id = 'test-org-smoke'" || true
+
+          # Create test billing account, organization and user first
+          echo "📝 Creating test billing account..."
+          BILLING_RESULT=$(wrangler d1 execute rushomon --remote -c wrangler.ephemeral.toml --command \
+            "INSERT INTO billing_accounts (id, owner_user_id, tier, created_at) \
+             VALUES ('test-org-smoke', 'test-user-smoke', 'free', $NOW)" --json)
+          echo "Billing account insert result: $(echo "$BILLING_RESULT" | jq '.[0].meta.changes // 0') changes"
+
           echo "📝 Creating test organization..."
           ORG_RESULT=$(wrangler d1 execute rushomon --remote -c wrangler.ephemeral.toml --command \
-            "INSERT INTO organizations (id, name, slug, created_at, created_by) 
-             VALUES ('test-org-smoke', 'Smoke Test Org', 'smoke-test', $NOW, 'test-user-smoke')" --json)
+            "INSERT INTO organizations (id, name, slug, created_at, created_by, billing_account_id) \
+             VALUES ('test-org-smoke', 'Smoke Test Org', 'smoke-test', $NOW, 'test-user-smoke', 'test-org-smoke')" --json)
           echo "Organization insert result: $(echo "$ORG_RESULT" | jq '.[0].meta.changes // 0') changes"
 
           echo "📝 Creating test user..."

